### PR TITLE
change module for steps to be inside contexts

### PR DESCRIPTION
**Pull request by rands0n**
What do you think of maintaining the `Step`s sections inside the father modules?

Also, this PR improve Shipment docs.

### CODE REVIEW

**Comment by rands0n** [comment]
This PR also partially solves the issue https://github.com/wesleimp/vroom-elixir/issues/1

File: lib/vroom/shipment_step.ex [hunk 1]
```diff
@@ -1,4 +1,4 @@
-defmodule VROOM.ShipmentStep do
+defmodule VROOM.Shipment.Step do
```
Comment (wesleimp) [inline lib/vroom/shipment_step.ex:1]: I'd suggest creating a new folder named `shipment` and move this file over there and rename it just to `step.ex`

File: lib/vroom/vehicle.ex [hunk 2]
```diff
@@ -19,7 +19,7 @@ defmodule VROOM.Vehicle do
           time_window: list() | nil,
           breaks: [VROOM.Break.t()] | nil,
           speed_factor: float(),
-          steps: [VROOM.VehicleStep.t()] | nil
+          steps: [VROOM.Vehicle.Step.t()] | nil
```
Comment (wesleimp) [inline lib/vroom/vehicle.ex:35]: The same comment about `shipment`

**Review by wesleimp — COMMENTED** [review]
Thanks, @rands0n

My only concern is when you alias those modules as it could confuse between `Vehicle.Step` and `Shipment.Step`

**Comment by rands0n** [comment]
@wesleimp Agreed, I've updated the files.

**Pull request merged**